harmonic/ros2_integration.md

```diff
@@ -1,16 +1,17 @@
 # ROS 2 Integration
 
 In this tutorial we will learn how to Integrate ROS 2 with Gazebo. We will establish
-communication between them. This can help in many aspects; we can receive data or commands
-from ROS and apply it to Gazebo and vice versa.
+communication between them. This can help in many aspects; we can receive data (like joint states, TFs) or commands
```

Comment:
Just to double-check, this file doesn't need to be updated in Fortress? There's no diff for this file in Fortress.

Reply:
So only `ros2` branch of ros_gz has this functionality, initially I thought Fortress isn't compatible but Fortress + Rolling would work. Will add that tidbit in all and the tutorial to Fortress.

Reply:
Fortress is the official version shipped with ROS 2 Humble (I believe it ships with desktop-full), and it's the [recommended](https://gazebosim.org/docs/harmonic/ros_installation) Gazebo version for Humble. Both Fortress and Humble are LTS, so I would hope that this works...

Reply:
Okay I confirmed this exists with `humble` branch as well, https://github.com/gazebosim/ros_gz/blob/humble/ros_gz_sim_demos/launch/sdf_parser.launch.py will work!

Reply:
Oh but it's not in `ros_ign_gazebo_demos`, so as per the ros_gz compatibility table it'll work with Rolling only. Is it okay if we don't block this for now, I won't make further additions? I'll do some tests or backward ports if required to ensure the recommended combinations work. I'll update the issue to include this.

Reply:
Yeah that's fine. Thank you for looking into it!